src/components/FiltersPanel/FiltersPanel.js

```diff
@@ -0,0 +1,134 @@
+            type="checkbox"
+            checked={filterState.catagoryName.flag}
+            onChange={() => filterDispatch({ type: "CATEGORY_NAME", payload: "flag" })}
+          />
+          Flag
+        </label><br/>
+        <label>
+          <input
+            type="checkbox"
+            checked={filterState.catagoryName.scaleModel}
+            onChange={() => filterDispatch({ type: "CATEGORY_NAME", payload:  "scaleModel"})}
+          />
+          Scale Model
+        </label><br/>
+        <label>
+          <input
+            type="checkbox"
+            checked={filterState.catagoryName.wearable}
+            onChange={() => filterDispatch({ type: "CATEGORY_NAME", payload: "wearable" })}
+          />
+          Wearable
+        </label><br/>
+               </form>
+            </div>
+            <div className="p-2 test-outline">
+               <h3>Rating</h3>
+               <form className="my-2">
+          <label >
+            <input
+              type="radio"
+              id="rating-opt-1"
+              name="option1"
+              value={4}
+              onChange={(e) => filterDispatch({type:"INPUT_RATING" , payload: Number(e.target.value) })}
+            />
+            4 stars and above
+          </label>
+          <br />
+          <label >
+            <input
+              type="radio"
+              id="rating-opt-2"
+              name="option1"
+              value={3}
+              onChange={(e) => filterDispatch({type:"INPUT_RATING" , payload: Number(e.target.value) })}
+            />
+            3 stars and above
+          </label>
+          <br />
+          <label >
+            <input
+              type="radio"
+              id="rating-opt-3"
+              name="option1"
+              value={2}
+              onChange={(e) => filterDispatch({type:"INPUT_RATING" , payload: Number(e.target.value) })}
+            />
+            2 stars and above
+          </label>
+          <br />
+          <label >
+            <input
+              type="radio"
+              id="rating-opt-4"
+              name="option1"
+              value={1}
+              onChange={(e) => filterDispatch({type:"INPUT_RATING" , payload: Number(e.target.value) })}
+            />
+            1 stars and above
+          </label>
+          <br />
+        </form>
+            </div>
+            <div className="p-2 test-outline">
+               <h3>Price</h3>
+               <form>
+          <label >
+            <input
+              type="radio"
+              name="option2"
+              onChange={() => filterDispatch({type:"INPUT_SORT", payload:"HIGH_TO_LOW"})}
+            />
+            High to low
+          </label>
+          <br />
+          <label >
+            <input
+              type="radio"
+              name="option2"
+              onChange={() => filterDispatch({type:"INPUT_SORT", payload:"LOW_TO_HIGH"})}
+            />
+            Low to high
+          </label>
+          <br />
```

Comment:
Avoid using br tags. Instead wrap them inside a div.

Reply:
Thanks for the suggestions and your time, I will rectify the code.